README.md

```diff
@@ -64,38 +63,49 @@ tox -e dev
 
 ```json
 {
-    "python.testing.unittestEnabled": false,
-    "python.testing.nosetestsEnabled": false,
-    "python.testing.pytestEnabled": true,
-    "pythonTestExplorer.testFramework": "pytest",
-    "python.testing.pytestArgs": [
-        "unittests"
-    ],
-    "python.linting.pylintEnabled": true
+  "python.testing.unittestEnabled": false,
+  "python.testing.nosetestsEnabled": false,
+  "python.testing.pytestEnabled": true,
+  "pythonTestExplorer.testFramework": "pytest",
+  "python.testing.pytestArgs": ["unittests"],
+  "python.linting.pylintEnabled": true
 }
 ```
+
 4. Create a `.env` file and insert the following line
 
 For Windows:
+
 ```
 PYTHONPATH=src;${PYTHONPATH}
 ```
+
 For Linux and Mac:
+
 ```
 PYTHONPATH=src:${PYTHONPATH}
 ```
+
 This makes sure, that the imports are working for the unittests.
 At the moment I am not totally sure that it is the best practise, but it's getting the job done.
 
 5. Enjoy 🤗
 
 ## Publishing on PyPI
+
 This repository contains all necessary CI steps to publish any project created from it on PyPI.
+It uses the trusted publishers workflow as described in the [official Python documentation](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/).
 It just requires some manual adjustments/settings depending on your project:
+
 1. Fill out the metadata in the [`pyproject.toml`](pyproject.toml); Namely the package name and the dependencies which should be in sync with your `requirements.in`.
 2. Uncomment the lines in [`.github/workflows/python-publish.yml`](.github/workflows/python-publish.yml)
-3. In [your PyPI account create a new API token](https://pypi.org/manage/account/#api-tokens). You have to create a token valid for your entire account first, only when the initial push happened, you can create a new token whose scope is limited to this project.
-4. Copy the token and paste it as a new repository secret under `github.com/your-username/your-reponame/settings/secrets/actions/new`. The secrets name should be `PYPI_API_TOKEN` as in the last line of the workflow file you edited in step 2.
+3. Create a new environment in your GitHub repository and call it `release`.
```

Comment:
maybe specify where? I don't remember seeing this setting anywhere. maybe add a link with placeholders like we did for release?

Reply:
fixed in f8765d9286404805f613c12cd62118863b7926bb